### PR TITLE
Allow the header/navbar to be fixed to the top of the page.

### PR DESCRIPTION
**Pull request by VizuaaLOG**
As requested by issue #46 this push adds the ability for fixing the header or navbar to the top of the page.

This push adds:
.header.is-fixed-top
.navbar.is-fixed-top

### CODE REVIEW

File: bulma/components/navbar.sass [hunk 1]
```diff
@@ -7,6 +7,13 @@
       margin-bottom: 10px
 
 .navbar
+  &.is-fixed-top
+    position: fixed
+    left: 0
+    right: 0
+    z-index: 1030
```
Comment (artyomtrityak) [inline bulma/components/navbar.sass]: why 1030? Bumping z-index will move to `z-index: 100000` issue

File: bulma/components/navbar.sass [hunk 2]
```diff
@@ -7,6 +7,13 @@
       margin-bottom: 10px
 
 .navbar
+  &.is-fixed-top
```
Comment (artyomtrityak) [inline bulma/components/navbar.sass]: should it have different behavior for mobile?

**Comment by jgthms** [comment]
The header is a good idea, but the navbar has no background so it shouldn't be "fixable" at the top.

I might rename that navbar actually.

File: bulma/components/navbar.sass (hunk 1 above)
Comment (VizuaaLOG) [inline bulma/components/navbar.sass]: There isn't a reason, I just chose 1030. What z-index issue?

**Comment by jgthms** [comment]
Can you create a new PR just for the header fixed on top? This PR got a bit crazy :astonished: 

And maybe add a class like `has-fixed-header` to the html so that the content doesn't end up behind the header?

**Comment by jbrooksuk** [comment]
Yes please!

**Comment by Bshally** [comment]
What is the implementation status of this PR? Interested in using Bulma for a project that requires a sticky header and would like to use an official component rather than creating my own if possible.

**Comment by mayurtolani** [comment]
So how do you use this fixed navbar property? It does not display anything on the bulma website.

**Comment by jgthms** [comment]
The source repository does not exist anymore.

**Pull request closed without merging**